lnbits/extensions/admin/crud.py

```diff
@@ -0,0 +1,64 @@
+from typing import Optional
+
+from lnbits.core.crud import create_payment
+from lnbits.helpers import urlsafe_short_hash
+from lnbits.settings import Settings, read_only_variables, settings
+from lnbits.tasks import internal_invoice_queue
+
+from . import db
+from .models import AdminSettings, UpdateSettings
+
+
+async def update_wallet_balance(wallet_id: str, amount: int):
+    internal_id = f"internal_{urlsafe_short_hash()}"
+
+    payment = await create_payment(
+        wallet_id=wallet_id,
+        checking_id=internal_id,
+        payment_request="admin_internal",
+        payment_hash="admin_internal",
+        amount=amount * 1000,
+        memo="Admin top up",
+        pending=False,
+    )
+    # manually send this for now
+    await internal_invoice_queue.put(internal_id)
+
+    return payment
+
+
+async def get_settings() -> AdminSettings:
+    row = await db.fetchone("SELECT * FROM admin.settings")
+    all_settings = Settings(**row)
+    settings = AdminSettings()
+    for key, value in row.items():
+        if hasattr(settings, key):
+            setattr(settings, key, getattr(all_settings, key))
+    return settings
+
+
+async def update_settings(data: UpdateSettings) -> Optional[Settings]:
+    fields = []
+    for key, value in data.dict(exclude_none=True).items():
```

Comment:
- extract `for` logic in separate function in `helpers.py`, keep the `crud` code ligt